File: lists.py
#Python does not use array by default it uses list

#Empty list
my_list = []    

day_list = ["Monday", "Tuesday",
            "Wednesday", "Thursday", 
            "Friday", "Saturday", "Sunday"]

#Accessing list items
print(day_list[2])

#Lists are itterable
for i in day_list:  
    print(i)                   #Returns the list item
    print(day_list.index(i))   #To get the index  


day_list.append("Crazyday") #Adds a last item
print(day_list)

day_list.pop(len(day_list)-1)    #Removes last item
print(day_list)

day_list.pop(0)    #Removes first item
print(day_list)

File: lists.js
//JS does not use list by default it uses array

//Empty array
myArray = [];

day_array = ["Monday", "Tuesday",
            "Wednesday", "Thursday", 
            "Friday", "Saturday", "Sunday"];

//Accessing array items            
console.log(day_array[2]);

//Arrays are itterable
for (i=0;i<day_array.length;i++){
    console.log(i)  //Returns the array index
    console.log(day_array[i])   //To get the item
}

day_array.push("Crazyday"); //Adds a last item
console.log(day_array);

day_array.pop();     //Removes last item
console.log(day_array);

day_array.shift();  //Removes first item
console.log(day_array);  